app/controllers/api/users_controller.rb

```diff
@@ -0,0 +1,21 @@
+module API
```

Comment:
This is just a temporary controller. I don't know whether we want the avatar attached to the user or the profile.